packages/salesforcedx-vscode-core/src/commands/orgDelete.ts

```diff
@@ -30,15 +30,15 @@ export class ForceOrgDeleteExecutor extends SfdxCommandletExecutor<{}> {
 
   public build(data: { choice?: string; username?: string }): Command {
     const builder = new SfdxCommandBuilder()
-      .withDescription(nls.localize('force_org_delete_default_text'))
-      .withArg('force:org:delete')
-      .withArg('--noprompt')
-      .withLogName('force_org_delete_default');
+      .withDescription(nls.localize('org_delete_default_text'))
+      .withArg('org:scratch')
```

Comment:
``` suggestion
      .withArg('org:delete:scratch')
```

Reply:
I wonder how tests were passing with this typo... 🤔

Reply:
Integration tests disabled.